### DISCUSSION TITLE
I have forked the project and have newer fixes

### DISCUSSION THREAD

**Discussion by wmerkens** [joedevivo/vscode-circuitpython#152] [General]
I have decided to fork this project to do the fixes myself and push a newer version to the VSC Marketplace. The repo is https://github.com/wmerkens/vscode-circuitpython, and on the VSC marketplace, look for wmerkens/circuitpython V2. My current stable version is V0.2.5, and I have a test version, V0.2.6-test, that incorporates a newer board selection list; I modified how it builds that list.

**Comment by IrregularShed** [comment]
Will, you've answered one of my wishes. I'm going to go and delete the fork that I tried hacking around at with no success. (Another wish is that Adafruit would sponsor the work on it, because I don't want to use any lesser editors!)

@joedevivo I know that you're online and working on other things from your user contribution chart, so I guess you've fallen out of love with CircuitPython. It happens, I know. But seeing as Adafruit have promoted the @wmerkens fork in their newsletter, could you come back just long enough to point potential users to the fork at the top of the readme? I'm certain that the broken state of v0.2.0 is damaging to new CP users.
